### PR TITLE
Generic summarisation script based on HF prompt template

### PR DESCRIPTION
**Pull request by sbroadhurst-hf**

### CODE REVIEW

File: summarize/01_summarize_transcripts_generic.py
Comment (faya98) [inline summarize/01_summarize_transcripts_generic.py:1]: Really liked the way it was structed and modularized. It was easy to read and understand the steps.

File: summarize/02_summarize_transcripts_generic_directory.py
Comment (faya98) [inline summarize/02_summarize_transcripts_generic_directory.py:1]: The script works really good.

Not a major concern. 
01_summarize_transcripts_generic.py script can produce empty summaries by skipping openai api call using "--dummy"
This script allows empty summaries to be processed. When the unlabelled file is uploaded the empty summaries also gets uploaded which are of no use.
Could check for those cases and handle them.

Comment (sbroadhurst-hf) [reply summarize/02_summarize_transcripts_generic_directory.py:1]: Good spot - extended row["skip"] to skip the api call and the file write.

**Review by faya98 — APPROVED** [review]
Nice work and liked the addition of model shift based on the number of input prompt tokens

**Review by faya98 — APPROVED** [review]
Nice work and liked the addition of model shift based on the number of input prompt tokens

**Pull request merged**